iio/iio.c

```diff
@@ -1247,6 +1247,8 @@ static uint32_t iio_generate_device_xml(struct iio_device *device, char *name,
  * @param dev_descriptor - Device descriptor
  * @param name - Name to identify the registered device
  * @param dev_instance - Opened instance of the device
+ * @param read_buff - read buffer
```

Comment:
The changes in this file should be subject for another commit.

Reply:
I was thinking it fits in the same commit because they were just small changes so the build doesn't fail, but I can make a separate commit just with fixes in general